.github/scripts/start_dd_agent.sh

```diff
@@ -49,12 +49,4 @@ fi
 
 # Apply changes
 sudo service datadog-agent stop
-
-INSTALL_DIR="/opt/datadog-agent"
-AGENTPATH="$INSTALL_DIR/bin/agent/agent"
-PIDFILE="$INSTALL_DIR/run/agent.pid"
-AGENT_ARGS="run -p $PIDFILE"
-AGENT_USER="dd-agent"
-LD_LIBRARY_PATH="/usr/local/cuda/extras/CUPTI/lib64:/usr/local/cuda/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64"
-
-sudo -E start-stop-daemon --start --background --quiet --chuid $AGENT_USER --pidfile $PIDFILE --user $AGENT_USER --startas /bin/bash -- -c "LD_LIBRARY_PATH=$LD_LIBRARY_PATH $AGENTPATH $AGENT_ARGS"
+sudo service datadog-agent restart
```

Comment:
`start-stop-daemon` was used directly to pass the `LD_LIBRARY_PATH` to the agent process, otherwise, GPU metrics won't be collected.